### PR TITLE
use different terminal ID

### PR DESCRIPTION
**Pull request by meganrogge**

### CODE REVIEW

File: src/vs/workbench/contrib/tasks/browser/terminalTaskSystem.ts [hunk 1]
```diff
@@ -120,7 +120,7 @@ class VariableResolver {
 	}
 }
 
-export class VerifiedTask {
+class VerifiedTask {
```
Comment (meganrogge) [inline src/vs/workbench/contrib/tasks/browser/terminalTaskSystem.ts:123]:
```suggestion
export class VerifiedTask {
```

**Review by rebornix — APPROVED** [review]

File: src/vs/workbench/contrib/tasks/browser/terminalTaskSystem.ts (hunk 1 above)
Comment (Mathishard0) [reply src/vs/workbench/contrib/tasks/browser/terminalTaskSystem.ts]: > ```suggestion
> export class VerifiedTask {
> ```

Comment (Mathishard0) [reply src/vs/workbench/contrib/tasks/browser/terminalTaskSystem.ts]: Ultimately

Comment (Mathishard0) [reply src/vs/workbench/contrib/tasks/browser/terminalTaskSystem.ts]: > ```suggestion
> export class VerifiedTask {
> ```

Comment (Shawnblake) [reply src/vs/workbench/contrib/tasks/browser/terminalTaskSystem.ts]: Nice

**Review by Cant2 — APPROVED** [review]

**Pull request merged**